src/containers/Login/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, FormEvent } from "react";
+import {
+    FormGroup,
+    ControlLabel,
+    FormControl,
+    Button,
+    FormControlProps
+} from "react-bootstrap";
+import "./Login.css";
+
+import { Auth } from "aws-amplify";
+
+const Login: React.FC = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    function validateForm() {
+        return email.length > 0 && password.length > 0;
+    }
+
+    function handleChange(event: FormEvent<FormControlProps & FormControl>) {
+        const { name, value } = event.target as HTMLInputElement;
+        switch (name) {
+            case "email":
+                setEmail(value);
+                break;
+            case "password":
+                setPassword(value);
+                break;
+        }
+    }
+
+    async function handleSubmit(e: FormEvent<HTMLFormElement>) {
+        e.preventDefault();
+        try {
+            await Auth.signIn(email, password);
+            alert("Logged in");
+        } catch (e) {
+            alert(e.message);
+        }
+    }
+
+    return (
+        <div className="Login">
+            <form onSubmit={handleSubmit}>
+                <FormGroup controlId="email" bsSize="large">
+                    <ControlLabel>Email</ControlLabel>
+                    <FormControl
+                        autoFocus
+                        type="email"
+                        name="email"
+                        onChange={handleChange}
+                    />
+                </FormGroup>
+                <FormGroup controlId="password" bsSize="large">
+                    <ControlLabel>Password</ControlLabel>
+                    <FormControl
+                        name="password"
+                        type="password"
+                        onChange={handleChange}
+                    />
+                </FormGroup>
+                <Button block bsSize="large" disabled={!validateForm()} type="submit">
+                    Login
+        </Button>
```

Comment:
Looks like you are formatting manually? Otherwise this should not be possible haha